cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/FileSystemUtil.java

```diff
@@ -72,6 +75,18 @@ public static FileSystem getOrCreateFileSystem(URI fileSystemUri) {
     return configFileSystem;
   }
 
+  /**
+   * @param properties path to a properties file
+   * @param output string to be written in the properties
+   * @throws FileNotFoundException
+   */
+  public static void writeOutputToFile(Path properties, String output) throws FileNotFoundException {
+
+    try (PrintStream printer = new PrintStream(new FileOutputStream(properties.toString()))) {
+      printer.print(output);
```

Comment:
Close the stream (this is also going to automatically flush the buffer)

Reply:
try with resources will close the stream automatically.